guides/advanced_guides/tokenization.md

```diff
@@ -0,0 +1,31 @@
+# Tokenization
```

Comment:
```suggestion
# Tokenization and Language Handling
```
Maybe a way to make it easier to understand for users that do not know the word tokenization? This is really just a suggestion. But I feel like you really have to know about the concept to know this word. In which case you wouldn't have to go in this guide cause you already know the concept.
And for users that do not know about tokenization, I suppose they will not have a reason to go on this guide. Except if it is linked somewhere that creates a context.

Reply:
100% yes :) It's such a nice feeling when someone suggests a change you were already making remotely :)

Reply:
Please, feel free to commit :)